Adyen/UI/Form/Items/Value/FormValueDetailItemView.swift

```diff
@@ -0,0 +1,189 @@
+        alertLabel.isAccessibilityElement = false
+        alertLabel.numberOfLines = 0
+        alertLabel.text = item.validationFailureMessage
+        alertLabel.accessibilityIdentifier = item.identifier.map { ViewIdentifierBuilder.build(scopeInstance: $0, postfix: "alertLabel") }
+        alertLabel.isHidden = true
+        
+        return alertLabel
+    }()
+    
+    // MARK: - Selection
+    
+    @objc
+    private func selectionButtonTapped() {
+        item.selectionHandler?()
+    }
+    
+    // MARK: - Convenience
+    
+    private func setupObservers() {
+        itemObserver = observe(item.publisher) { [weak self] _ in
+            self?.updateValidationStatus()
+        }
+        
+        observe(item.$formattedValue) { [weak self] in
+            self?.updateValueLabel(with: $0)
+        }
+        
+        observe(item.$validationFailureMessage) { [weak self] in
+            self?.alertLabel.text = $0
+        }
+    }
+    
+    private func updateValueLabel(with formattedValue: String?) {
+        guard let formattedValue, !formattedValue.isEmpty else {
+            valueLabel.text = item.placeholder
+            valueLabel.textColor = item.style.placeholderText?.color ?? .Adyen.componentPlaceholderText
+            return
+        }
+        
+        valueLabel.text = formattedValue
+        valueLabel.textColor = item.style.text.color
+    }
+    
+    override open func configureSeparatorView() {
+        let constraints = [
+            separatorView.leadingAnchor.constraint(equalTo: layoutMarginsGuide.leadingAnchor),
+            separatorView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            separatorView.heightAnchor.constraint(equalToConstant: 1.0)
+        ]
+        
+        NSLayoutConstraint.activate(constraints)
+    }
+    
+    private func configureConstraints() {
+        selectionButton.adyen.anchor(inside: self)
+        separatorView.bottomAnchor.constraint(equalTo: valueLabel.bottomAnchor, constant: 4).isActive = true
+    }
+    
+    // MARK: - Validation
+    
+    override public func validate() {
+        updateValidationStatus(forced: true)
+    }
+    
+    private func updateValidationStatus(forced: Bool = false) {
+        if !item.isValid(), forced {
+            hideAlertLabel(false)
+            highlightSeparatorView(color: item.style.errorColor)
+            titleLabel.textColor = item.style.errorColor
+            valueLabel.accessibilityLabel = item.validationFailureMessage
+        } else {
+            hideAlertLabel(true)
+            unhighlightSeparatorView()
+            titleLabel.textColor = defaultTitleColor
+            valueLabel.accessibilityLabel = item.title
+        }
+    }
+    
+    private func hideAlertLabel(_ hidden: Bool, animated: Bool = true) {
+        guard hidden || alertLabel.text != nil else { return }
+        alertLabel.adyen.hide(animationKey: "hide_alertLabel", hidden: hidden, animated: animated)
```

Comment:
Would be quite useful to move "hide_alertLabel" into an enum so you don't have to deal with strings in the middle of the code.